doc/bitfileDeployment.md

```diff
@@ -0,0 +1,161 @@
+
+<img src="./images/labviewRealTimeSystem.png" width="300" height="300" />
+
+### Field Programmable Gate Arrays (FPGA)
+
+FPGAs are silicon chips with unconnected logic gates. 
+You can define the functionality of the FPGA by using software to configure the FPGA gates. 
+FPGAs are completely reconfigurable and instantly take on a brand new “personality” when you download a different configuration of circuitry.
+
+Common use of FPGA are high-performance control loops, high-speed signal acquisition and generation, signal processing and analysis, safe state logic, etc.
+
+### Programming the FPGA With LabVIEW
+
+The FPGA circuitry is configured using the NI LabVIEW FPGA Module.
+The first step is develop a FPGA VI in LabVIEW to define the FPGA logic.
+You can download and run only one top-level FPGA VI at a time on a single FPGA target. 
+If you download a second top-level VI to the FPGA target, the second VI overwrites the first VI.
+
+After the FPGA VI is developed, you want to execute the VI on
+the FPGA target, which will require compiling of code.
+
+### Compiling the FPGA VI
+
+The LabVIEW FPGA Module compiles VIs to FPGA hardware using an automatic, multi-step process. 
+Behind the scenes, the VI is translated to text-based **V**ery-High-Speed Integrated Circuit **H**ardware **D**escription **L**anguage (VHDL) code.
+Then industry-standard Xilinx ISE compiler tools optimize, reduce, and synthesize the VHDL code into a hardware circuit realization of the LabVIEW design. 
+During the FPGA compilation, the VI is optimized to reduce digital logic and create an optimal implementation of the LabVIEW application. 
+The end result is a `bitfile` (.lvbitx) that contains binary data that describes how to configure the FPGA circuit of the FPGA target. 
+When the application runs, the bitfile reconfigures the FPGA circuit of the FPGA target. 
+Go [here](https://www.ni.com/docs/en-US/bundle/labview-fpga-module/page/lvfpgaconcepts/compiling_fpga_vis.html) to know more details about the compilation process.
+
+Each bitfile is unique to the type of the FPGA target for wich the bitfile was compiled.
+
+After we have this bitfile just created, is time to download it into the CompactRIO and create this new "personality".
+
+## How to deploy the bitfile into the CompactRIO?
+
+Following are the available options to download the bitfile into the CompactRIO.
+
+### 1. Using NI LabVIEW Real-Time Module
+
+The LabVIEW Real-Time Module is a LabVIEW add-on you can use to create reliable, stand-alone applications that execute on embedded hardware devices like a CompactRIO.
+
+The most prevalent method for deploying an FPGA personality (bitfile) is to embed it into the host application. 
+This inclusion into the application occurs when the `Open FPGA Reference` LabVIEW function is used in the host implementation. 
+When the host application is then compiled into an executable (`.rtexe`) the FPGA application is embedded inside this file. 
```

Comment:
After the host application is compiled into

Reply:
Done.